src/components/accordion-item/accordion-item.tsx

```diff
@@ -190,12 +220,15 @@ export class AccordionItem implements ConditionalSlotComponent {
               role="button"
               tabindex="0"
             >
-              {this.icon ? iconEl : null}
-              <div class={CSS.headerText}>
-                <span class={CSS.heading}>{this.heading || this.itemTitle}</span>
-                {this.itemSubtitle || this.description ? (
-                  <span class={CSS.description}>{this.description || this.itemSubtitle}</span>
-                ) : null}
+              <div class={CSS.headerContainer}>
+                {this.iconStart ? iconStartEl : null}
```

Comment:
nitpick: would be nice to move the conditional part to where the const is being declared.